src/test/java/myCollections/MyLinkedListTest.java

```diff
@@ -0,0 +1,58 @@
+package myCollections;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MyLinkedListTest {
+
+    @Test
+    public void checkAdd() {
+        MyLinkedList linkedList = new MyLinkedList<String>();
+        linkedList.add("ss");
+        String actual = (String)linkedList.get(0);
```

Comment:
Why you should cast this element. There are some problems with implementation. Please fix them.